validators/cgroup_validator_linux.go

```diff
@@ -44,10 +47,31 @@ func (c *CgroupsValidator) Name() string {
 }
 
 const (
-	cgroupsConfigPrefix = "CGROUPS_"
-	unifiedMountpoint   = "/sys/fs/cgroup"
+	cgroupsConfigPrefix      = "CGROUPS_"
+	defaultUnifiedMountpoint = "/sys/fs/cgroup"
+	mountsFilePath           = "/proc/mounts"
 )
 
+// getUnifiedMountpoint will check /proc/mounts and return the cgroup dir
+func getUnifiedMountpoint() string {
+	f, err := os.Open(mountsFilePath)
+	if err != nil {
+		fmt.Printf("error checking %q: %v\n", mountsFilePath, err)
+		return defaultUnifiedMountpoint
+	}
+
+	defer f.Close()
+	scanner := bufio.NewScanner(f)
+	for scanner.Scan() {
+		// example fields: `cgroup2 /sys/fs/cgroup cgroup2 rw,seclabel,nosuid,nodev,noexec,relatime 0 0`
+		fields := strings.Split(scanner.Text(), " ")
```

Comment:
Just in case, I think that strings.Fields is suitable to handle spaces like below:
https://github.com/kubernetes/system-validators/blob/1d2f5889f07347b9f2d3ed337b3cfb08d45f4cf6/validators/cgroup_validator_linux.go#L132

Reply:
Updated.